components/director/internal/domain/webhook/repository.go

```diff
@@ -44,7 +46,7 @@ func NewRepository(conv EntityConverter) *repository {
 	return &repository{
 		singleGetter: repo.NewSingleGetter(resource.Webhook, tableName, tenantColumn, webhookColumns),
 		creator:      repo.NewCreator(resource.Webhook, tableName, webhookColumns),
-		updater:      repo.NewUpdater(resource.Webhook, tableName, []string{"type", "url", "auth"}, tenantColumn, []string{"id", "app_id"}),
+		updater:      repo.NewUpdater(resource.Webhook, tableName, updatableColumns, tenantColumn, []string{"id", "app_id"}),
```

Comment:
As there are now more than just `id` and `app_id`, but also `runtime_id` and `integration_system` is this `idColumns` enough?.
We have to keep in mind that if in the future we want to have for example `runtime_id` and not `app_id` the update SQL will now work as it will add where clause with `app_id`

Reply:
not sure if this is enough descriptive, but you can ask offline to further discuss. I am not seeing it as a problem as of now, but perhaps in the future

Reply:
As a second thought, each of them can be empty right? If this is true, then there will only be where clause with empty `app_id` for example. Then you can just add the rest of the ID fields in this array for completeness or leave only the `id` of the webhook. Not sure why the `app_id` is needed at all though.